### ISSUE TITLE
Django-Rest-Framework Support 

### ISSUE THREAD

**Issue by nasir733** [pennersr/django-allauth#3004]

**Comment by nasir733** [comment]
Plzz Add support for django restframework

**Comment by peter-stuart-turner** [comment]
It doesn't seem clear to me how to use this package with DRF? I think having that clearer would bring a bunch of people onto this package

**Comment by gschurck** [comment]
I agree, there is [dj-rest-auth package](https://github.com/iMerica/dj-rest-auth), but it's not very clear to find how to user django-allauth social authentication with it. The main functionalities of django-allauth seems to be mainly implemented in views with forms. Could be very useful to have also serializers for DRF and dj-rest-auth.

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3004
